hack/verify-ends-newline.sh

```diff
@@ -25,6 +25,7 @@ function check_ends() {
     -o -iname "*.sh" \
     -o -iname "*.go" \
     -o -iname "*.yaml" \
+    -o -iname "*.tpl" \
```

Comment:
I think it would be better to solve this problem with a new PR

Reply:
ok

Reply:
@wzshiming I tested it and when `docker exec -it kwok-kwok-control-plane mv /etc/kubernetes/manifests/kube-controller-manager.yaml /etc/kubernetes/manifests/kube-controller-manager.yaml.bak` is executed, it deletes the old pod and starts a new pod
```sh
➜  ~ k -n kube-system get po
NAME                                              READY   STATUS    RESTARTS   AGE
coredns-558bd4d5db-c4sxf                          0/1     Pending   0          15m
coredns-558bd4d5db-x7wlq                          0/1     Pending   0          15m
etcd-kwok-kwok-control-plane                      1/1     Running   0          15m
kindnet-97jkb                                     1/1     Running   0          13m
kindnet-zrtg6                                     1/1     Running   0          15m
kube-apiserver-kwok-kwok-control-plane            1/1     Running   0          15m
kube-controller-manager-kwok-kwok-control-plane   1/1     Running   0          15m
kube-proxy-2f2mg                                  1/1     Running   0          15m
kube-proxy-hclf7                                  1/1     Running   0          13m
kube-scheduler-kwok-kwok-control-plane            1/1     Running   0          15m
kwok-controller-kwok-kwok-control-plane           1/1     Running   0          15m
➜  ~ docker exec -it kwok-kwok-control-plane mv /etc/kubernetes/manifests/kube-controller-manager.yaml /etc/kubernetes/manifests/kube-controller-manager.yaml.bak
➜  ~ k -n kube-system get po
NAME                                              READY   STATUS    RESTARTS   AGE
coredns-558bd4d5db-c4sxf                          0/1     Pending   0          17m
coredns-558bd4d5db-x7wlq                          0/1     Pending   0          17m
etcd-kwok-kwok-control-plane                      1/1     Running   0          17m
kindnet-97jkb                                     1/1     Running   0          16m
kindnet-zrtg6                                     1/1     Running   0          17m
kube-apiserver-kwok-kwok-control-plane            1/1     Running   0          17m
kube-controller-manager-kwok-kwok-control-plane   1/1     Running   0          29s
kube-proxy-2f2mg                                  1/1     Running   0          17m
kube-proxy-hclf7                                  1/1     Running   0          16m
kube-scheduler-kwok-kwok-control-plane            1/1     Running   0          17m
kwok-controller-kwok-kwok-control-plane           1/1     Running   0          17m
```